SOQL Builder GA doc update (#3422)
forcedotcom/salesforcedx-vscode: docs/_articles/en/soql/soql-builder.md

---
title: SOQL Builder (Beta)

---
title: SOQL Builder
lang: en
---

> SOQL Builder is a VS Code extension that eliminates the guesswork when building SOQL queries. With SOQL Query Builder, anyone can visually build, run, and explore results from queries. Build queries using clicks in a visual editor, then save and extend the queries using a text editor. You can instantly view query results, and then save the results to a `.csv` or `.json` file.

Reviewer: Is the official product name SOQL Query Builder? If so, I'd also change the title above.

## Using SOQL Builder

> You can your SOQL Builder to build simple query statements that include:

Reviewer: ```suggestion
Use SOQL Builder to build simple query statements that include:
```

- LIMIT clause

  To dig deeper regarding SOQL syntax or to build more complex queries in the text editor, see the [SOQL and SOSL Reference guide](https://developer.salesforce.com/docs/atlas.en-us.soql_sosl.meta/soql_sosl/sforce_api_calls_soql_sosl_intro.htm).
  
**Note**
- You can run complex queries in SOQL Builder even if you see the Unsupported Syntax informational message.
- WHERE clauses can be quite complex. SOQL Builder supports simple WHERE expressions. You can combine conditions using AND or OR, but not both.

### Set Up

> - Make sure you are authenticated into your default org. You can’t use SOQL builder if authentication to default org has expired.

Reviewer: ```suggestion
- Make sure you are authenticated into your default org. You can’t use SOQL Builder if authentication to the default org has expired.
```

<!-- **Tip:** If using the text editor to build your query, you can validate your syntax by turning on the SOQL Editor Remote Checks setting. ekapner update, 2/2/2021: this setting not ready for GA-->

**Limitations:**

> - SOQL Builder currently supports interactively building simple queries. We plan to add more functionality soon. However, you can still open a more complex `.soql` file and run the query from within SOQL Builder, but you must use a text editor to update it.

Reviewer 2: @sbudhirajadoc - Remove "We plan to add more functionality soon."
